### PR TITLE
Support UTF-8 input action names

### LINKED ISSUE #26380: Godot doesn't recognize actions with unicode character
**Godot version:**
3.1 beta8

**Issue description:**
When you name any action with unicode character (e.g. "góra"), methods like `Input.is_action_pressed("góra")` will always return false, regardless if the key is pressed.

**Minimal reproduction project:**
[TestProject.zip](https://github.com/godotengine/godot/files/2912498/TestProject.zip)

### PR DESCRIPTION
**Pull request by ptrojahn**
Fixes #26380

### CODE REVIEW

File: core/variant_parser.cpp [hunk 1]
```diff
@@ -1542,6 +1542,9 @@ Error VariantParser::parse_tag_assign_eof(Stream *p_stream, int &line, String &r
 			} else if (c != '=') {
 				what += String::chr(c);
 			} else {
+				if(p_stream->is_utf8()){
```
Comment (akien-mga) [inline core/variant_parser.cpp]: Style issue, see http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

**Comment by reduz** [comment]
this is good

**Review by reduz — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**